src/post/post.repository.knex.ts

```diff
@@ -150,6 +150,12 @@ export class KnexPostRepository {
         });
     }
 
+    async getByUuid(uuid: Post['uuid']): Promise<Post | null> {
```

Comment:
I don't think we need this, we can just get by ap id instead? Once we lose the link between ghost post uuid and ap post uuid, this becomes redundant anyway

Reply:
Ghost webhook req doesn't have the ap_id, do you mean we generate ap_id using uuid and then fetch using it?